app/geant-exporter/BremsstrahlungProcess.hh

```diff
@@ -0,0 +1,68 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2021 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file BremsstrahlungProcess.hh
+//! \brief Bremsstrahlung process class.
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <string>
+#include <G4VEnergyLossProcess.hh>
+
+namespace geant_exporter
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Bremsstrahlung model selection.
+ */
+enum class BremsModelSelection
```

Comment:
Since this doesn't need to be accessed by a template class (or indeed anything else that lacks access to BremssProcess) I would rename it to `ModelSelection` and embed it inside the class itself.